test/module/irohad/consensus/yac/yac_gate_test.cpp

```diff
@@ -112,8 +118,13 @@ TEST_F(YacGateTest, YacGateSubscriptionTest) {
 
   // verify that yac gate emit expected block
   auto gate_wrapper = make_test_subscriber<CallExact>(gate->on_commit(), 1);
-  gate_wrapper.subscribe(
-      [this](auto block) { ASSERT_EQ(block, expected_block); });
+  gate_wrapper.subscribe([this](auto block) {
```

Comment:
may be rework with shared_model?
I think expected_block should be shared_model.